src/main/java/org/generationcp/middleware/manager/GermplasmDataManagerImpl.java

```diff
@@ -1590,4 +1596,14 @@ public long countAttributesByGid(final String gid, final List<String> attributeD
 		return this.getAttributeDao().getByIDs(ids);
 	}
 
+	@Override
+	public List<KeySequenceRegister> getKeySequenceRegistersByPrefixes(final List<String> keyPrefixes) {
```

Comment:
I think we should not add these two methods in `GermplasmDataManager` but instead be accessed via `keySequenceRegisterService` from webapps.